pom.xml

```diff
@@ -69,12 +69,6 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.mockito</groupId>
-            <artifactId>mockito-all</artifactId>
-            <version>1.9.5</version>
-            <scope>test</scope>
-        </dependency>
```

Comment:
`spring-boot-starter-test` already comes with mockito... and this dep is not compbatible with it.